crates/sp-domains-fraud-proof/src/lib.rs

```diff
@@ -188,3 +190,18 @@ impl FraudProofVerificationInfoResponse {
         }
     }
 }
+
+sp_api::decl_runtime_apis! {
+    /// API necessary for fraud proof.
+    #[api_version(2)]
```

Comment:
Why do we need to have the `api_version` to 2?
This api is not introduced in the current gemini-3g runtime.